includes/class-create-block-theme-api.php

```diff
@@ -144,6 +144,18 @@ public function register_rest_routes() {
 				},
 			),
 		);
+		register_rest_route(
+			'create-block-theme/v1',
+			'/reset-theme',
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => array( $this, 'rest_reset_theme' ),
+				// 'permission_callback' => function () {
+				// 	return current_user_can( 'edit_theme_options' );
+				// },
```

Comment:
Can we remove this comment or should we add the permission check?

Reply:
thanks for noticing that, the comment was for testing something locally. It wasn't plan to be commited. It removed it now:  [`8ca3269` (#687)](https://github.com/WordPress/create-block-theme/pull/687/commits/8ca3269c2a42d47c973889017a20632164d87b43)